entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -561,10 +678,67 @@ class EntityHandlerTests {
             )
     }
 
+    @Test
+    fun `get entity by id should show relationship sysAttrs when asked`() {
```

Comment:
Maybe you can change the title to smth like:
get entity by id should include relationship temporal properties if optional query param sysAttrs is present

Reply:
OK